docs/source/index.rst

```diff
@@ -0,0 +1,41 @@
+Flask-Ninja
+===================================
+
+**Flask-Ninja** is a simple web API framework based on Flask.
+It uses Pydantic for validation and automatic generation of Openapi documentation. It's heavily inspired by `django-ninja <https://django-ninja.rest-framework.com/>`__ framework.
```

Comment:
```suggestion
It uses Pydantic for validation and automatic generation of OpenAPI documentation. It's heavily inspired by `django-ninja <https://django-ninja.rest-framework.com/>`__ framework.
```

as mentioned above, try to unify the naming of tools and features mentioned across these docs